src/darwin/Framework/CHIPTests/TestHelpers/MTRTestCase+ServerAppRunner.m

```diff
@@ -51,9 +51,6 @@ + (NSTask *)doStartAppWithName:(NSString *)name arguments:(NSArray<NSString *> *
     __auto_type * appTask = [self createTaskForPath:executable];
 
     __auto_type * forcedArguments = @[
-        // Make sure we only advertise on the local interface.
```

Comment:
Again, this is going to break the Darwin tests, causing a lot of random failures due to cross-talk between runners.  Stopping here.